merkledag/merkledag_test.go

```diff
@@ -115,12 +116,12 @@ func SubtestNodeStat(t *testing.T, n *Node) {
 	}
 
 	expected := NodeStat{
+		Hash:           k.B58String(),
 		NumLinks:       len(n.Links),
-		BlockSize:      len(enc),
-		LinksSize:      len(enc) - len(n.Data), // includes framing.
+		BlockSize:      bs,
+		LinksSize:      ls,
```

Comment:
links size just refers to the size of the marshalled link objects, not the entire size of the dag below this node.